lib/internal/test_runner/mock.js

```diff
@@ -157,18 +161,40 @@ class MockTracker {
         'options.setter', setter, "cannot be used with 'options.getter'"
       );
     }
+    function getOriginalObject(descriptor) {
```

Comment:
It looks like this is only used in one place. Maybe inline the code like it previously was to avoid creating a closure on every call to `method()`.

Reply:
added!